packages/bitcore-node/src/services/p2p.ts

```diff
@@ -279,10 +285,64 @@ export class P2pService {
     }
     logger.info(`${chain}:${network} up to date.`);
     this.syncing = false;
-    StateModel.collection.findOneAndUpdate({}, {$addToSet: { initialSyncComplete: `${chain}:${network}`}}, { upsert: true});
+    StateModel.collection.findOneAndUpdate(
+      {},
+      { $addToSet: { initialSyncComplete: `${chain}:${network}` } },
+      { upsert: true }
+    );
     return true;
   }
 
+  async createFork(newChain: string, parentChain: string, forkHeight: number) {
+    const blockCursor = BlockModel.collection.find({ parentChain, height: { $lt: forkHeight } }, { batchSize: 100 });
```

Comment:
will want to get localtip for the child chain add add as part of the parentChain block Cursor Query - that way it can be restarted in middle of process